app/collins/models/Asset.scala

```diff
@@ -266,7 +266,7 @@ object Asset extends Schema with AnormAdapter[Asset] with AssetKeys {
     val findLocations = find(PageParams(0, 50, "ASC", "tag"), AttributeResolver.emptyResultTuple, instanceFinder)
       .items
       .collect { case a: Asset => a }
-      .filter { _.tag != MultiCollinsConfig.thisInstance }
+      .filter(asset => { !MultiCollinsConfig.thisInstance.contains(asset.tag) })
```

Comment:
If you think the code has an "obvious" function, then OK, else a comment would be nice.